WordPress/Classes/ViewRelated/Reader/ReaderSearchSuggestionsViewController.swift

```diff
@@ -0,0 +1,182 @@
+import UIKit
+import WordPressShared
+
+///
+///
+protocol ReaderSearchSuggestionsDelegate
+{
+    func searchSuggestionsController(controller: ReaderSearchSuggestionsViewController, selectedItem: String)
+    func clearSuggestionsForSearchController(controller: ReaderSearchSuggestionsViewController)
+}
+
+
+///
+///
+class ReaderSearchSuggestionsViewController : UIViewController
+{
+    @IBOutlet var stackView: UIStackView!
+    @IBOutlet var tableView: UITableView!
+    @IBOutlet var clearButton: UIButton!
+    @IBOutlet var borderImageView: UIImageView!
+    @IBOutlet var stackViewHeightConstraint: NSLayoutConstraint!
+
+    var phrase = "" {
+        didSet {
+            updatePredicateAndRefresh()
+            updateHeightConstraint()
+        }
+    }
+
+    var tableViewHandler: WPTableViewHandler!
+    var delegate: ReaderSearchSuggestionsDelegate?
+    var cellIdentifier = "CellIdentifier"
+    let MaxTableViewRows = 5
+    let ButtonHeight = CGFloat(44.0)
```

Comment:
Q: did we settle on capitalized constants, or those should start with lowercase?